src/components/Portfolio.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import { Card, CardGroup} from "react-bootstrap";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import "./Portfolio.css";
+//Assets

```

Comment:
This comment should have a space before the word `Assets` to be consistent with [this one](https://github.com/InaiaK/React-website/pull/3/files#diff-8bc8933bf9f0a1dece53f0af6a1cf667e51f3286cc1d8aacebe87ff459e2960bR3)